vaadin-date-time-picker-flow-demo/src/main/java/com/vaadin/flow/component/datetimepicker/demo/DateTimePickerView.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2000-2019 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.datetimepicker.demo;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
+
+import com.vaadin.flow.component.datetimepicker.DateTimePicker;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.Label;
+import com.vaadin.flow.component.html.NativeButton;
+import com.vaadin.flow.demo.DemoView;
+import com.vaadin.flow.router.Route;
+
+/**
+ * View for {@link DateTimePicker} demo.
+ *
+ * @author Vaadin Ltd
+ */
+@Route("vaadin-date-time-picker")
+public class DateTimePickerView extends DemoView {
+
+    @Override
+    public void initView() {
+        createSimpleDateTimePicker();
+        createLocaleChangeDateTimePicker();
+        addCard("Additional code used in the demo",
+                new Label("These methods are used in the demo."));
+    }
+
+    private void createSimpleDateTimePicker() {
+        Div message = createMessageDiv("simple-picker-message");
+
+        // begin-source-example
+        // source-example-heading: Simple date time picker
+        DateTimePicker dateTimePicker = new DateTimePicker();
+
+        dateTimePicker.addValueChangeListener(
+                event -> updateMessage(message, dateTimePicker));
+        // end-source-example
+
+        dateTimePicker.setId("simple-picker");
+        addCard("Simple date time picker", dateTimePicker, message);
+    }
+
+    private void createLocaleChangeDateTimePicker() {
+        Div message = createMessageDiv("Customize-locale-picker-message");
+        // begin-source-example
+        // source-example-heading: Date picker with customize locales
```

Comment:
I see this is copied from date-picker demos, but "with customize locales" doesn't sound grammatically correct.

Maybe "with custom locales"? Although I don't know whats so "custom" about these locales.

Reply:
I Updated it to "with different locales".

Reply:
Nice 👍